playwright.config.ts

```diff
@@ -0,0 +1,83 @@
+import { defineConfig, devices } from '@playwright/test';
+
```

Comment:
I'm leaving a lot of this commented code in atm as some of it will become relevant when it comes to running these tests in CI / GH Actions.